src/main.js

```diff
@@ -5,7 +5,7 @@ new Vue({
   render: (h) => {
     return h(App, {
       props: {
-        variation: 'resource',
+        variation: 'location',
```

Comment:
Leftover from dev or did you change the default intentionally?

Reply:
Yep, leftover. Will roll it back

Reply:
Done